packages/client/src/components/MediaCard.tsx

```diff
@@ -0,0 +1,74 @@
+import AddIcon from '@mui/icons-material/Add';
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+
+type CardProps = {
+  name: string;
+  content: string;
+  img: {
+    url: string;
+    alt: string;
+  };
+  price: number;
+  calories: {
+    perPortion: number;
+  };
+  fats: {
+    perPortion: number;
+  };
+  proteins: {
+    perPortion: number;
+  };
+  carbohydrates: {
+    perPortion: number;
+  };
+};
+
+const MediaCard = ({ name, content, img, price, calories, fats, proteins, carbohydrates }: CardProps) => {
+  return (
```

Comment:
```suggestion
type CardProps = {
  dish: Omit<DishDto, 'id' | 'restaurant'>;
};

const MediaCard = ({ dish }: CardProps) => {
  const { name, description, photo, fats, proteins, carbohydrates, calories, price } = dish;
  return (
```

Lepiej będzie wykorzystać `DishDto` (zaimportowany `import { DishDto } from '@fullstack/sdk';`) bo:
- jak później dania będą szły z bazy a nie przykładowe, to będą to właśnie `DishDto`
- tworzymy teraz drugi praktycznie identyczny typ, który trzeba będzie aktualizować równolegle z dowolnymi zmianami Dish
- trzeba by potem dodatkowo bawić się w zamianę obiektów `DishDto` na `CardProps`.

Możesz później to w `MediaCardGrids` wywołać tak:
```TSX
<MediaCard
  dish={{
    name: 'Bowl japoński',
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ',
    photo: cardImg,
    price: 32,
    calories: { perPortion: 550, per100g: 0 },
    fats: { perPortion: 250, per100g: 0 },
    proteins: { perPortion: 20, per100g: 0 },
    carbohydrates: { perPortion: 20, per100g: 0 },
  }}
/>
```

Co później zmieni się w:
```TSX
<MediaCard dish={dishFromDB} />
```

Reply:
super ! właśnie tak czułam, że się "powtarzam" w kodzie, ale nie mogłam tego wszystkiego połączyć :D